lib/charms/layer/apache_bigtop_base.py

```diff
@@ -39,12 +40,32 @@ def site_yaml(self):
     def __init__(self):
         self.bigtop_dir = '/home/ubuntu/bigtop.release'
         self.options = layer.options('apache-bigtop-base')
-        self.bigtop_version = self.options.get('bigtop_version')
-        self.bigtop_apt = self.options.get(
-            'bigtop_repo-{}'.format(utils.cpu_arch()))
-        self._bigtop_base = Path(self.bigtop_dir) / self.bigtop_version
-        self._site_yaml = self.bigtop_base / self.options.get(
-            'bigtop_hiera_siteyaml')
+        self.bigtop_version = self.options['bigtop_version']
+        self._bigtop_base = Path(self.bigtop_dir) / 'bigtop-{}'.format(
+            self.bigtop_version)
+        self._site_yaml = Path(self.bigtop_base) / self.options[
+            'bigtop_hiera_siteyaml']
```

Comment:
@johnsca, this silently failed for me:

```
        self.bigtop_apt = self.options.get(
            'bigtop_repo-{}'.format(utils.cpu_arch()))
```

because cpu_arch returns 'ppc64le' on power, and we had the option set as 'ppc64el', so `.get('nonexistent')` just came back blank.